<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>421</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="기본">
     <property name="whatsThis">
      <string>123</string>
     </property>
     <attribute name="title">
      <string>기본 업무</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>토요일 폴더</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>일요일 폴더</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>241</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>복붙</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>카톡보고</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>오늘 날짜</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>자동화</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>수업정보 복붙</string>
      </property>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>기본 양식</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>토 10시 수업</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>토 1시 수업</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>일 1시 수업</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>일 3시10분</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>링크</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>110</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>스프레드 시트</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>맞춤법 검사</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>110</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>ACA 웹</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>보강 캘린더</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>341</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>보고 파일 이름 변경</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>폴더 열기</string>
      </property>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>보고파일3개</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>수업자료</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="README">
     <attribute name="title">
      <string>README</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>391</width>
        <height>371</height>
       </rect>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>380</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Readme</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="카카오톡피드백">
     <attribute name="title">
      <string>카카오톡 피드백</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>391</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>510</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>숙제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>510</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>카톡폴더</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
